src/pages/home/components/productMountainSlider/productMountainSlider.tsx

```diff
@@ -0,0 +1,114 @@
+import { useEffect, useRef } from 'react';
+import styles from './productMountainSlider.module.scss';
+import RecommendedProducts from '../recommendedProducts/recommendedProducts';
+
+const RADIUS = 1200;
+const FLIP_RANGE = 3;
+
+interface newProduct {
+  productImages: { url: string }[];
+  details?: string;
+  id: string;
+  category: string;
+  ratings?: number;
+  price: number;
+  bonus: number;
+  collectionId: string;
+  createdAt?: string;
+  expDate: string;
+  expiredflag: boolean;
+  name: string;
+  quantity: number;
+}
+
+interface ProductsProps {
+  imageData: newProduct[];
+}
```

Comment:
does `imageData` have this `newProduct` instances? (category,rating,.....)
may be you can change the name of `imageData` to give it an understandable meaning

Reply:
it is fine, they are the same